public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,84 @@
+# dvc.api.get_url()
+
+Return the URL
+([string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)
```

Comment:
let's not include types here

Reply:
Yep. I still need to implement the change from **Signature** to **Definition** and include all these technicalities there per https://github.com/iterative/dvc.org/pull/908#pullrequestreview-348442563 Working on it now ⏳

Reply:
OK implemented Definition sections in all functions (and matching updates in iterative/dvc/pull/3352). Please check